### DISCUSSION TITLE
Need to change Control name in Views when command name changed

### DISCUSSION THREAD

**Discussion by rulai-yinqiang** [harborsiem/WindowsRibbon#26] [General]
Include Size definitions and Tab.
For example: In Ribbon Tools, when I change the command name from 'cmd_1' to 'cmd_2', that command in Views still remain 'cmd_1', if I changed some names, it's not easy to find all the name changed commands in views.

**Comment by harborsiem** [comment]
Can't reproduce.
When I changed the command name with "Enter" button after changing, I see in Views the new command name.
When I change without "Enter", in Views there is no binding to a command name.

**Comment by rulai-yinqiang** [comment]
Sorry, it seems Size Definitions is just a map. I also used command names in it. And it will not change. Now, I understand, it's only a map, don't need change. This is not a problem any more.
